enos/scripts/test_e2e_env.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+DIR=$(pwd)
+
+SCRIPTS_DIR=$( cd -- "$( dirname -- "$0" )" &> /dev/null && pwd )
+STATEDIR=$(ls -td $SCRIPTS_DIR/../.enos/*/ | head -1) # get latest directory
+
+cd $STATEDIR
+terraform show -json terraform.tfstate | jq -r '.values.root_module.child_modules[].resources[] | select(.address=="module.run_e2e_test.enos_local_exec.run_e2e_test") | .values.environment | to_entries[] | "export \(.key)=\(.value|@sh)"'
+terraform show -json terraform.tfstate | jq -r '.values.root_module.child_modules[].resources[] | select(.address=="module.run_e2e_ui_test.enos_local_exec.run_e2e_ui_test") | .values.environment | to_entries[] | "export \(.key)=\(.value|@sh)"'
```

Comment:
The thing I like about this script is that it will output something like
```
...
export E2E_SSH_PORT='22'
export E2E_SSH_USER='ubuntu'
export E2E_TARGET_IP='10.13.12.185'
...
```
and I can just copy and paste that into a terminal session.